marketplace_builder/views/pages/api-reference/liquid/flow-control.liquid

```diff
@@ -0,0 +1,185 @@
+---
+converter: markdown
+slug: api-reference/liquid/flow-control
+searchable: true
+metadata:
+  title: API reference - Liquid Flow Control
+---
+
+## If / Else
+if / else statements should be familiar from other programming languages. Liquid implements them with the following tags:
+
+* {% raw %}`{% if <CONDITION> %} ... {% endif %}`{% endraw %} — Encloses a section of template which will only be run if the condition is true.
+* {% raw %}`{% elsif <CONDITION> %}`{% endraw %} — Can optionally be used within an `if` ... `endif` block. Specifies another condition; if the initial "if" fails, Liquid tries the "elsif", and runs the following section of template if it succeeds. You can use any number of elsifs in an if block.
+* {% raw %}`{% else %}`{% endraw %} — Can optionally be used within an `if` ... `endif` block, after any "elsif" tags. If all preceding conditions fail, Liquid will run the section of template following the "else" tag.
+* {% raw %}`{% unless <CONDITION> %} ... {% endunless %}`{% endraw %} — The reverse of an "if" statement. Don't use "elsif" or "else" with an unless statement.
+
+The condition of an if, elsif or unless tag should be either a normal Liquid expression or a comparison using Liquid expressions.
+Note that the comparison operators are implemented by the "if"-like tags; they don't work anywhere else in Liquid.
+
+## Operators
+
+The available comparison operators are:
+
+* `==`, `!=`, `and` `<>` — equality and inequality (the latter two are synonyms)
+* `<`, `<=`, `>`, `>=` — less/greater-than
+* There's a secret special value `empty` (unquoted) that you can compare arrays to; the comparison is `true` if the array has no members.
+* contains — similar to Ruby `include?` method, implemented on strings, arrays, and hashes. If the left argument is a string and the right isn't, it stringifies the right.
+
+## Boolean operators
+
+The available Boolean operators are:
+
+* `and`
+* `or`
+
+You can use the and and or operators to include more than one condition in a control flow tag. and and or can be chained together to create complex conditionals.
+
+If you use multiple and or or operators, note that and operators will be evaluated first, then or operators. You cannot use parentheses to simulate an order of operations and control the order of operator evaluation. Parentheses are invalid characters within Liquid tags and prevent your tags from working.
+
+### and
+
+The and operator lets you add additional conditions to a tag. A condition with an and will only be true if both the left and the right side of the condition are true.
```

Comment:
We might want to explicitly state that the very common operators `&&` and `||` will NOT work but will not raise errors